ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/usertags/cucushift-installer-rehearse-azure-ipi-usertags-workflow.yaml

```diff
@@ -3,6 +3,7 @@ workflow:
   steps: 
     pre:
     - chain: cucushift-installer-rehearse-azure-ipi-provision
+    - chain: cucushift-installer-check-azure
```

Comment:
This chain can be extended to all the install workflow?

Reply:
yeah, those chain can be run on most azure ipi workflows. But there is a rate to hit below prow system error on step, in order to make less noise for CI job, how about to only run this chain in some workflows?

```
INFO[2023-10-27T17:58:18Z] Running step aws-ipi-sdn-mtu-migrate-f28-network-mtu-migration-decrease-end. 
INFO[2023-10-27T18:28:18Z] pod pending for more than 30m0s: containers have not started in 30m0.000147894s: ci-scheduling-dns-wait, cp-entrypoint-wrapper, place-entrypoint, sidecar, test: 
* Container sidecar is not ready with reason PodInitializing
* Container test is not ready with reason PodInitializing 
```

Reply:
>  how about to only run this chain in some workflows

Understood, okay for me.

Reply:
Beside `cucushift-installer-rehearse-azure-ipi-usertags`, are we going to extend it to some other install workflow, or in our current plan, this is the only one.

Reply:
yeah, I just added chain into install chains `cucushift-installer-rehearse-azure-ipi-sdn` and `cucushift-installer-rehearse-azure-mag-ipi` to see if it works well on those workflows.